<launch>
  <node name="cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name ="cup_detector_node" pkg="cup_detector" type="cup_detector.py" output="screen" required="true"/>
</launch>
